docs/src/examples/IEGT_example1.jl

```diff
@@ -0,0 +1,91 @@
+# # Foundations of Bilevel Programming: Example 9
+# This example is from the book Investment in Electricity Generation and Transmission. [url](https://www.springer.com/gp/book/9783319294995)
+
+# Bold point(s): using Dualof() in case we have information on the dual variable(s) of the lower level problem. 
```

Comment:
I don't understand this line.

Reply:
I add the important points of the example. Better to say, what makes it different from other examples, and here is the usage of the Dualof() command in the model.